pkg/clients/edgeconnect/client.go

```diff
@@ -87,8 +87,8 @@ type ServerError struct {
 
 // DetailsError represents details of errors
 type DetailsError struct {
-	ConstraintViolations ConstraintViolations `json:"constraintViolations"`
-	MissingScopes        []string             `json:"missingScopes,omitempty"`
+	ConstraintViolations []ConstraintViolations `json:"constraintViolations"`
```

Comment:
is it something unrelated to this PR?

Reply:
Related - I couldn't understand server's response related to invalid k8s hostname. It seems the structure definition has been incorrect since the very beginning.